src/modsim2/utils/config.py

```diff
@@ -35,6 +35,42 @@ def load_configs(
     return config_dict
 
 
+def compose_transform(
```

Comment:
as discussed in person, I think we should expect users to specify the full list of transforms required (including any ToPil or ToTensor). Let's put this in the config instead (maybe using variables so we don't end up repeating huge strings)

Reply:
Have done this!